pkg/cassandra/config/config.go

```diff
@@ -211,6 +273,23 @@ func (c *Configuration) String() string {
 }
 
 func (c *Configuration) Validate() error {
+	govalidator.CustomTypeTagMap.Set("cassandraTTLValidation", func(i interface{}, _ interface{}) bool {
```

Comment:
why do we need custom functions, can the conditions not be expressed directly via tags?

Reply:
if we must have custom logic then just write it directly, with proper error messages - right now you are just returning a Boolean, so at best validator can say "property x did not validate".

Reply:
> why do we need custom functions, can the conditions not be expressed directly via tags?

Couldn't find such validations here: https://github.com/asaskevich/govalidator

> if we must have custom logic then just write it directly, with proper error messages - right now you are just returning a Boolean, so at best validator can say "property x did not validate".

Fixed it.